src/actions/guides/frontend/internationalization.cr

```diff
@@ -0,0 +1,526 @@
+class Guides::Frontend::Internationalization < GuideAction
+  guide_route "/frontend/internationalization"
+
+  def self.title
+    "Internationalization"
+  end
+
+  def markdown : String
+    <<-MD
+    **Working with multiple languages**
+
+    If these steps are done in oder then Lucky should continue to compile (& be usable/testable) with each change.
+
+    **Summary:**
+    After configuration you can apply translations using either:
+    ```
+    # simplest (requires user_lang or current_user to be defined)
+    t("tranlation.key.values")
+
+    # or where ever current user is available
+    I18n.t("tranlation.key.values", current_user.lang)
+    ```
+
+    > This document assumes you are using the default Authorization - if not you will need to make adjustments to the `user` in the Translator!
```

Comment:
```suggestion
    > This document assumes you are using the default [Authentication](https://luckyframework.org/guides/authentication) - if not, you will need to make adjustments to the `user` in the Translator!
```

Note: the link to the authentication page should actually follow how we link to other pages internally. There's a few steps that has to be taken for that.